custom_code/templatetags/photometry_extras.py

```diff
@@ -40,11 +41,15 @@ def recent_photometry(context, target, limit=1):
     recent_det = {'data': []}
     for datum in datums.order_by('-timestamp')[:limit]:
         if 'magnitude' in datum.value.keys():
-            recent_det['data'].append({'timestamp': datum.timestamp, 'magnitude': datum.value['magnitude']})
+            phot_point = {'timestamp': datum.timestamp, 'magnitude': datum.value['magnitude']}
         elif 'limit' in datum.value.keys():
-            recent_det['data'].append({'timestamp': datum.timestamp, 'limit': datum.value['limit']})
-        else:
-            continue
+            phot_point = {'timestamp': datum.timestamp, 'limit': datum.value['limit']}
+
+        if target.distance is not None:
```

Comment:
Just to check, will this also not run if the distance of the best galaxy is a nan? There aren't many hosts in GLADE/GWGC like this but does come up

Reply:
It will still run, but the absolute magnitude will also be nan.